src/main/java/edu/hm/hafner/analysis/FileLocation.java

```diff
@@ -0,0 +1,208 @@
+package edu.hm.hafner.analysis;
+
+import edu.umd.cs.findbugs.annotations.CheckForNull;
+
+import java.io.Serial;
+import java.io.Serializable;
+
+/**
+ * Represents a file location associated with an issue. This can be used to represent additional files that are related
+ * to a warning, such as header files referenced in C++ reorder warnings, or multiple files traced in execution paths
+ * for tools like Fortify and Coverity.
+ *
+ * @author Akash Manna
+ */
+public class FileLocation implements Serializable {
+    @Serial
+    private static final long serialVersionUID = 1L;
```

Comment:
```suggestion
    private static final long serialVersionUID = 6L; 
```

I try to use the same number for all serializations